### ISSUE TITLE
impossible opening

### ISSUE THREAD

**Issue by pasqal** [QGIS-Contribution/QGIS-ResourceSharing#260]
### What is the bug or the crash?

Hi, i can't use plugin , I'm on MX-linux 21.1

Error message :
"Impossible de charger l'extension 'qgis_resource_sharing' provoque une erreur lors de l'appel à sa méthode classFactory() 

ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/usr/lib/python3.9/site-packages/markupsafe/__init__.py) 
Traceback (most recent call last):
  File "/app/share/qgis/python/qgis/utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/__init__.py", line 54, in classFactory
    from qgis_resource_sharing.plugin import ResourceSharingPlugin
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/plugin.py", line 33, in 
    from qgis_resource_sharing.gui.resource_sharing_dialog import ResourceSharingDialog
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/gui/resource_sharing_dialog.py", line 42, in 
    from qgis_resource_sharing.collection_manager import (
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/collection_manager.py", line 13, in 
    from qgis_resource_sharing.repository_handler import BaseRepositoryHandler
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/repository_handler/__init__.py", line 1, in 
    from .base import BaseRepositoryHandler  # noqa: F401
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/repository_handler/base.py", line 14, in 
    from qgis_resource_sharing.network_manager import NetworkManager
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/network_manager.py", line 7, in 
    from qgis_resource_sharing.utilities import qgis_version
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/utilities.py", line 5, in 
    import jinja2
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/app/lib/python3.9/site-packages/jinja2/__init__.py", line 12, in 
    from .environment import Environment
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/app/lib/python3.9/site-packages/jinja2/environment.py", line 25, in 
    from .defaults import BLOCK_END_STRING
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/app/lib/python3.9/site-packages/jinja2/defaults.py", line 3, in 
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/app/share/qgis/python/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/app/lib/python3.9/site-packages/jinja2/filters.py", line 13, in 
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/usr/lib/python3.9/site-packages/markupsafe/__init__.py)

Thanks for your help.

### Steps to reproduce the issue

just try instal

### Versions

Version de Python : 3.9.9 (main, Nov 10 2011, 15:00:00) [GCC 11.2.0] 
Version de QGIS : 3.22.6-Białowieża Białowieża, exported 

Chemin Python :
/app/share/qgis/python
/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python
/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins
/app/share/qgis/python/plugins
/usr/lib/python39.zip
/usr/lib/python3.9
/usr/lib/python3.9/lib-dynload
/usr/lib/python3.9/site-packages
/app/lib/python3.9/site-packages
/app/lib/python3.9/site-packages/MarkupSafe-2.1.0-py3.9-linux-x86_64.egg
/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python
/home/pasqal/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins/qgis_resource_sharing/ext_libs"

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the official roadmap.

### New profile

- [X] I tried with a new QGIS profile

**Comment by Guts** [comment]
Hi,

Thanks for your interest and taking time to report.

About your report:

- would you mind to edit your issue to correctly format error log to make it easily readable please?
- could you specify which plugin version do you try to install please?

About your problem:

- would it not be a duplicate of #243?

**Comment by pasqal** [comment]
I tried 0.16.0 and 1.0.0
Jinja2 and markupsafe are up to date

$ python3 -m pip show markupsafe jinja2
Name: MarkupSafe
Version: 2.1.1
Summary: Safely add untrusted strings to HTML/XML markup.
Home-page: https://palletsprojects.com/p/markupsafe/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD-3-Clause
Location: /home/pasqal/.local/lib/python3.9/site-packages
Requires: 
Required-by: Jinja2
---
Name: Jinja2
Version: 3.1.2
Summary: A very fast and expressive template engine.
Home-page: https://palletsprojects.com/p/jinja/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD-3-Clause
Location: /home/pasqal/.local/lib/python3.9/site-packages
Requires: MarkupSafe
Required-by:
